BaseTools/Conf/tools_def.template

```diff
@@ -988,9 +988,9 @@ DEFINE GCC5_X64_CC_FLAGS             = DEF(GCC49_X64_CC_FLAGS)
 DEFINE GCC5_IA32_X64_DLINK_COMMON    = DEF(GCC49_IA32_X64_DLINK_COMMON)
 DEFINE GCC5_IA32_X64_ASLDLINK_FLAGS  = DEF(GCC49_IA32_X64_ASLDLINK_FLAGS)
 DEFINE GCC5_IA32_X64_DLINK_FLAGS     = DEF(GCC49_IA32_X64_DLINK_FLAGS)
-DEFINE GCC5_IA32_DLINK2_FLAGS        = DEF(GCC49_IA32_DLINK2_FLAGS) -Wno-error
+DEFINE GCC5_IA32_DLINK2_FLAGS        = DEF(GCC49_IA32_DLINK2_FLAGS) -Wno-error -Wl,-u,gotassert
 DEFINE GCC5_X64_DLINK_FLAGS          = DEF(GCC49_X64_DLINK_FLAGS)
-DEFINE GCC5_X64_DLINK2_FLAGS         = DEF(GCC49_X64_DLINK2_FLAGS) -Wno-error
+DEFINE GCC5_X64_DLINK2_FLAGS         = DEF(GCC49_X64_DLINK2_FLAGS) -Wno-error -Wl,-u,gotassert

```

Comment:
Based on this commit https://github.com/tianocore/edk2/commit/01225075dbf my understanding is that GCC would become the rolling current toolchain, and GCC5 would possibly stay as is and become legacy at some point? cc @bexcran 

If so, again in my understanding and apologies if I'm missing something, the updates here would ideally be applied to both GCC and GCC5?

Reply:
Yes. Counter-intuitively, `GCC5_X64_DLINK2_FLAGS` is actually consumed to construct `*_GCC_X64_DLINK2_FLAGS` so this should work as expected.

Reply:
Ahh, right! I don't think I have an option to mark as resolved, but resolved, tyvm.